src/popper/runner_host.py

```diff
@@ -147,7 +147,7 @@ def run(self, step):
         if not container and not self._config.reuse:
             container = self._create_container(cid, step)
 
-        log.info(f"[{step.id}] docker start")
+        log.step_info(f"[{step.id}] docker start")
```

Comment:
these were OK as `log.info()`, this are supposed to still be shown when `--quiet` is given